lesson_09/homeworks.py

```diff
@@ -0,0 +1,70 @@
+def leftover_from_division(a,b):
+    """
+    Функція знаходження остачі від ділення
+    :param a: перше число
+    :param b: друге число
+    :return: остача від ділення
+    """
+    #print(f'Остача від ділення {a} на {b} дорівнює {a % b}, '
+    #      f'де {a} - {a // b} * {b} = {a % b}')
+    return a % b
+
+#print(leftover_from_division(8019,0))
+
+def if_any_sentence_begins_with_word (text: list, find_sentence: str):
+    """
+    Функція перевіряє чи починається якесь речення зі списку з необхідного тексту
+    :param text: список речень
+    :param find_sentence: початок речення по якому буде пошук
+    """
+    cout_test = sum([1 for test in text if test.startswith(find_sentence)])
+    #if cout_test >= 1:
```

Comment:
You need to avoid pushing code that is commented into final PR.

Reply:
Ok, will clean in new homework :)